_episodes/07-aliases.md

```diff
@@ -30,42 +30,25 @@ keypoints:
 
 ### Global aliases
 
-A simple shortcut:
+A very useful shortcut which we will use a lot today:
 
 ```shell
-$ git config --global alias.co checkout
-$ cd a_git_repo
-$ git co [branch_name]
-```
-
-A more useful shortcut:
-
-```shell
-$ git config --global alias.ls "log --graph --decorate --pretty=oneline"
-$ cd a_git_repo
-$ git ls
+$ git config --global alias.graph "log --all --graph --decorate --oneline"
```

Comment:
Do you think we should include all aliases in one big block that can be copied and pasted (instead of having to find the different blocks to copy and paste)?  Then below we could provide details and discuss them separately more.

I'm not decided yet if they should be like this (one block) or split up with different discussion.  I'm leaning towards the first because likely more people will grab and copy than read.

Reply:
I will leave it up to you to reformat - I have no strong opinion. As an intermediate participant I would probably not take the entire list but only the ones I like. As a total beginner I might copy the entire thing.